### PR TITLE
[zh-cn] sync tasks/administer-cluster/configure-upgrade-etcd.md

### PR DESCRIPTION
**Pull request by my-git9**
[zh-cn] sync tasks/administer-cluster/configure-upgrade-etcd.md

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 1]
```diff
@@ -608,11 +608,20 @@ ETCDCTL_API=3 etcdctl --endpoints 10.2.0.9:2379 snapshot restore snapshotdb
 
 <!--
 Another example for restoring using etcdctl options:
+
+Yet another example would be to first export the environment variable
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/configure-upgrade-etcd.md:612]:
```suggestion
Yet another example would be to first export the environment variable
```
This line should be moved before its trans

File: content/zh-cn/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 2]
```diff
@@ -608,11 +608,20 @@ ETCDCTL_API=3 etcdctl --endpoints 10.2.0.9:2379 snapshot restore snapshotdb
...
 -->
 恢复时也可以指定操作选项，例如：
 
 ```shell
-ETCDCTL_API=3 etcdctl --data-dir <data-dir-location> snapshot restore snapshotdb
+ETCDCTL_API=3 etcdctl snapshot restore --data-dir <data-dir-location> snapshotdb
+```
+
+另一个例子是先设置环境变量
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/configure-upgrade-etcd.md:620]:
```suggestion
另一个例子是先导出环境变量：
```

**Review by windsonsea — CHANGES_REQUESTED** [review]
It's not a good image for you if a small PR has too many issues there.

File: content/zh-cn/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 3]
```diff
@@ -612,7 +612,17 @@ Another example for restoring using etcdctl options:
...
 
 ```shell
-ETCDCTL_API=3 etcdctl --data-dir <data-dir-location> snapshot restore snapshotdb
+ETCDCTL_API=3 etcdctl snapshot restore --data-dir <data-dir-location> snapshotdb
+```
+
+<!--
+Yet another example would be to first export the environment variable
+-->
+另一个例子是先导出环境变量：
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/administer-cluster/configure-upgrade-etcd.md:621]:
```suggestion
另一个例子是先設定环境变量：
```

**Pull request merged**